tenable/sc/base.py

```diff
@@ -213,7 +213,7 @@ def _schedule_constructor(self, item):
         if resp['type'] == 'ical' and 'start' in item and 'repeatRule' in item:
             resp['start'] = self._check('schedule:start', item['start'], str)
             resp['repeatRule'] = self._check(
-                'schedule:repeatRule', item['repeatRule'], str)
```

Comment:
#144 specifies that the changes should be made to the documentation not to the code. the condition in line 213 now conflicts with the field that you are pulling out of `item`. The best way to fix this would be to change the documentation to match the code. see line 298 of `/tenable/sc/repositories.py` otherwise you need to also change all TSC references to `repeatRule`

Reply:
Good point. I have reverted the changes under issue 144. I agree that the preferred approach is to update the documentation as it seems it's the only thing out of sync.

Reply:
In that case just update the relevant docstrings.  My fault as IO && Nessus use rrule, whereas SC uses repeatRule.